src/app/services/listingmembre.service.ts

```diff
@@ -14,7 +14,15 @@ export class ListingmembreService {
         .get(`${url.API}/membres.json?enabled=1`)
           .map((res: Response) => {
             return res.json();
-          });
+                });
+  }
+  
+  getDatalazyLoad(sort : string , order: string, page: number){
+    return this.http
+    .get(`${url.API}/menbres.json?enabled=1&sort=${sort}&order=${order}&page=${page + 1}`)
+     .map((res: Response) => {
+      return res.json();
```

Comment:
indentation , and sort don't exist , its "pagination" and "page"  get parameters

Reply:
@tiboCorb fix this please

exemple : `&pagination=true&itemsPerPage=10`
btw if u don't specify theses parameters, pagination  ain't avalaible